stories/CssReset.stories.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import { storiesOf } from '@storybook/react'
 import { CssReset } from '../src'
 
+// eslint-disable-next-line react/prop-types
```

Comment:
I'm ignoring prop-types for storybook inline components, seems excessive to check those as well.